### DISCUSSION TITLE
training C api - model export from buffer session

### DISCUSSION THREAD

**Discussion by riccardopinosio** [microsoft/onnxruntime#21047] [API Q&A]
Hi everyone,

Currently it seems that if you create a training session with the C api using `CreateTrainingSessionFromBuffer` there is no way to export the final model using `ExportModelForInferencing` because the latter complains that no `eval` path has been provided.

I wonder if it would be possible to support exporting the final model from a session created from buffers?

**Comment by carzh** [comment]
Could you create a GitHub issue for this bug to track?

**Reply by riccardopinosio** [reply]
will do, if I have time I might even take a stab at fixing it

**Reply by carzh** [reply]
thanks!

**Comment by carzh** [comment]
Issue tracked in #21152
